source/draw/RasterMonochrome.ooc

```diff
@@ -224,4 +224,10 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		result
 	}
+	writePixelGpu: override func (x, y: Int, color: ColorRgba) {
```

Comment:
I would prefer not to have switch cases that fails at runtime since it is not modular but if they can be named in a better way that states behaviour explicitly then I am open for suggestions.